scripts/package.js

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const commander = require('commander');
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+const path = require('path');
+const replace = require('replace-in-file');
+
+function stageFiles(commander, outputDir, datFile) {
+  const baseDatFile = path.parse(datFile).base;
+
+  const originalManifest = path.join('manifests', commander.type + '-manifest.json');
+  const updatedManifest = path.join(outputDir, 'manifest.json');
+
+  const replaceOptions = {
+    files: updatedManifest,
+    from: /0\.0\.0/g,
+    to: commander.setVersion,
+  };
+
+  mkdirp.sync(outputDir);
+
+  fs.copyFileSync(originalManifest, updatedManifest);
+  fs.copyFileSync(datFile, path.join(outputDir, baseDatFile));
+
+  replace.sync(replaceOptions);
+}
+
+function generateCRX(commander, outputDir, datFile) {
+  const ChromeExtension = require('crx');
+  const crx = new ChromeExtension({
+    privateKey: fs.readFileSync(commander.key)
+  });
+
+  const crxFile = commander.type + '.crx';
+
+  crx.load(path.resolve(outputDir))
+    .then(() => crx.loadContents())
+    .then((archiveBuffer) => {
+      crx.pack(archiveBuffer).then((crxBuffer) => {
+        fs.writeFileSync(crxFile, crxBuffer);
+        console.log('Generated ' + crxFile);
+      });
+    })
+    .catch((err) => {
+      console.error(err.stack);
```

Comment:
Best to not ignore the error here as it'll get swallowed by CI when not run manually.
You can `throw err` after the `console.error` to make sure it is thrown.